src/main/webapp/i18n/de/guidedTourCourseExerciseOverview.json

```diff
@@ -3,7 +3,7 @@
         "courseExerciseOverview": {
             "installPrerequisites": {
                 "headline": "Installiere die notwendigen Tools",
-                "content": "Bevor wir uns den Kurs und die darin enthaltene Beispielaufgabe anschauen, müssen wir zuerst die notwendigen Tools herunterladen und installieren.<ul><li>Installiere das neueste <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (mind. Java JDK 16). Stelle dabei sicher, dass du das JDK und <span class='red bold'>nicht</span> JRE auswählst.</li><li>Installiere die <a href='https://www.eclipse.org/downloads/packages/' target='_blank' rel='noopener noreferrer'>Eclipse IDE für Java Entwickler</a> (2021-03 oder neuer)</li><li>Installiere die neueste Version von <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> mit git (4.1.0 oder neuer für macOS bzw. 3.4.1 oder neuer für Windows)</li></ul>",
+                "content": "Bevor wir uns den Kurs und die darin enthaltene Beispielaufgabe anschauen, müssen wir zuerst die notwendigen Tools herunterladen und installieren.<ul><li>Installiere das neueste <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (mind. Java JDK 16). Stelle dabei sicher, dass du das JDK und <span class='red bold'>nicht</span> JRE auswählst.</li><li>Installiere die <a href='https://www.eclipse.org/downloads/packages/' target='_blank' rel='noopener noreferrer'>Eclipse IDE für Java Programmierende</a> (2021-03 oder neuer)</li><li>Installiere die neueste Version von <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> mit git (4.1.0 oder neuer für macOS bzw. 3.4.1 oder neuer für Windows)</li></ul>",
```

Comment:
Did you checked that the german translation of eclipse is called "Eclipse IDE für Java Programmierende" should be the name of the program (or am I wrong)

Reply:
Furthermore, "Entwickler:in" != "Programmierer:in"